platform/android/export/export.cpp

```diff
@@ -1381,7 +1381,7 @@ Error EditorExportPlatformAndroid::export_project(const String& p_path, bool p_d
 
 		err = OS::get_singleton()->execute(jarsigner,args,true,NULL,NULL,&retval);
 		if (retval) {
-			EditorNode::add_io_error("'jarsigner' verificaiton of APK failed. Make sure to use jarsigner from Java 6.");
+			EditorNode::add_io_error("'jarsigner' verification of APK failed. Make sure to use jarsigner from Java 6.");
```

Comment:
This PR has been merged already so it can't be modified anymore. I was just letting you know for future PRs :) As long as it hasn't been merged, you can force push to your branch and the PR will be updated automatically.